samples/msal-browser-samples/HybridSample/README.md

```diff
@@ -0,0 +1,200 @@
+# Hybrid SPA sample application
+
+This sample demonstrates how to use MSAL.js v2 and MSAL Node together in a "hybrid" application that performs both server-side and client-side authenication. 
```

Comment:
Can we change this description to also reflect the ssoSilent working with an auth code without depending on a "hint"?